Library/Homebrew/cask/config.rb

```diff
@@ -54,7 +54,7 @@ def self.from_file(path)
     end
 
     def self.canonicalize(config)
-      config.map { |k, v| [k.to_sym, Pathname(v).expand_path] }.to_h
+      config.map { |k, v| [k.to_sym, DEFAULT_DIRS.key?(k.to_sym) ? Pathname(v).expand_path : v] }.to_h
```

Comment:
Feels like quite a lot for inside a `[]` and with a ternary operator. An if/else and intermediate variable would make this easier to follow.